source/kubernetes/user-access.rst

```diff
@@ -9,17 +9,17 @@ Introduction
 ************
 
 Kubernetes clusters launched on the Catalyst Cloud are integrated with the
-Identity and Access Management (IAM) service. Users with one of the roles
+OpenStack Keystone (Identity) service. Users with one of the roles
 listed below are able to interact with any Kubernetes clusters owned by their
 project using their existing cloud credentials.
 
-The IAM roles related to the Kubernetes service are:
+The OpenStack Keystone Identity roles related to the Kubernetes service are:
 
-* ``k8s-admin`` administrator of the cluster platform and able to perform all
+* ``k8s_admin`` administrator of the cluster platform and able to perform all
   operations within the cluster.
-* ``k8s-developer`` can deploy applications to the cluster but cannot perform
-  destructive operations within the ``kube-system`` namespace.
-* ``k8s-viewer`` can only have view and obtain information of cluster
+* ``k8s_developer`` can deploy applications to the cluster but cannot change cluster
```

Comment:
The wording here is awkward "but cannot change cluster level operations", perhaps "but cannot perform cluster level operations"?

Reply:
Agreed, I don't like the wording either. New commit has simplified and separated the language for your review.